src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import MenuItem from "@mui/material/MenuItem";
+import FormControl from "@mui/material/FormControl";
+import Select, { SelectChangeEvent, SelectProps } from "@mui/material/Select";
+import { boxStyle, selectStyle } from "./Filter-style";
+import { InputLabel } from "@mui/material";
+import { ReactComponent as DropDownIcon } from "../../assets/Icons/dropdown.svg";
+import { title } from "process";
+
+interface IFilter {
+  title: string;
+  items?: string[];
```

Comment:
Why it's optional?

Reply:
I did this to make it work without data, I'll delete it now